### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by AndresFMoya**
Hello! This project consist of build a sign up form, using nearly-pure HTML, then form_tag helper and finally form_for helper. The forms_tag, and html version are commented out.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  validates :username, presence: true
```
Comment (mosaaleb) [inline app/models/user.rb]: - [ ] **[OPTIONAL]** You can list multiple attributes if they share the same validator =>
```ruby
validates :username, :email, :password, presence: true
```

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to new_user_path
```
Comment (mosaaleb) [inline app/controllers/users_controller.rb]: - [ ] redirect_to edit_user_path if user was saved -- you also should provide the path generator with the user corresponding id. 
    `edit_user_path(@user)`

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Well done @KerronKing @AndresFMoya :clap:

#### You are nearly done with the requirements


- [x] `new.html.erb` and `edit.html.erb` have the same code which is repeatitive. We can DRY these views using [partial files](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials).


- [x] See my comments below.

#### Please handle these requirements and submit a new code review.

#### Happy Coding :computer:

**Comment by AndresFMoya** [comment]
Hi @mosaaleb. Thanks for your feedback. Changes were made, and we ask for a new code review. Regards!

**Review by mosaaleb — APPROVED** [review]
### Well done @AndresFMoya, @KerronKing :clap:

#### All requirements are handled. You can merge :+1: 

#### Good luck with the next projects. Happy Coding :computer:

**Pull request merged**